crates/flux-rustc-bridge/src/mir.rs

```diff
@@ -198,6 +198,7 @@ pub enum StatementKind {
     Nop,
 }
 
+/// Corresponds to https://doc.rust-lang.org/beta/nightly-rustc/rustc_middle/mir/enum.Rvalue.html
```

Comment:
Bare URLs are not automatically turned into clickable links

```suggestion
/// Corresponds to <https://doc.rust-lang.org/beta/nightly-rustc/rustc_middle/mir/enum.Rvalue.html>
```